Sources/TuistSupport/Extensions/String+Extras.swift

```diff
@@ -104,6 +104,11 @@ extension String {
         prefix(1).lowercased() + dropFirst()
     }
 
+    public var isURL: Bool {
+        let pattern = "(?i)https?://(?:www\\.)?\\S+(?:/|\\b)"
```

Comment:
This will reject other valid forms, such as SSH URLs (`git@github.com:jrescabias/cliTemplate.git`).

Reply:
@ezraberch thats true, just did modify this to cover all the options 😀

Reply:
[This](https://github.com/jonschlinkert/is-git-url/blob/master/index.js#L9) might be useful.